packages/oid4vci-demo-frontend/src/configs/banqup-identity.json

```diff
@@ -0,0 +1,249 @@
+            "key": "firstName",
+            "label": "ssi_information_request_page_form_first_name_label",
+            "type": "text",
+            "required": true
+          }
+        ],
+        [
+          {
+            "id": "ssi_information_request_page_form_last_name",
+            "key": "lastName",
+            "label": "ssi_information_request_page_form_last_name_label",
+            "type": "text",
+            "required": true
+          }
+        ],
+        [
+          {
+            "id": "ssi_information_request_page_form_email",
+            "key": "emailAddress",
+            "label": "ssi_information_request_page_form_email_label",
+            "type": "text",
+            "required": true,
+            "customValidation": "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$"
+          }
+        ],
+        [
+          {
+            "id": "ssi_information_request_page_form_bsn_number",
+            "key": "bsnNumber",
+            "label": "ssi_information_request_page_form_bsn_number_label",
+            "type": "text",
+            "required": true
+          }
+        ],
+        [
+          {
+            "id": "termsAndConditions",
+            "key": "termsAndConditions",
+            "label": "ssi_information_request_page_form_terms_and_conditions_label",
+            "labelUrl": "https://google.com",
+            "type": "checkbox",
+            "required": true,
+            "display": {
+              "checkboxBorderColor": "#202776",
+              "checkboxLabelColor": "#424242",
+              "checkboxSelectedColor": "#202776"
+            }
+          }
+        ]
+      ]
+    },
+    "issueRequest": {
+      "leftPaneWidth": "40%",
+      "title": "title_secure_your_personal_credential",
+      "topParagraph": "paragraph_add_credential",
+      "bottomParagraph": "paragraph_scan_this_qr",
+      "backgroundColor": "#0E0B31",
+      "logo": {
+        "src": "banqup/logo.svg"
+      },
+      "mobile": {
+        "logo": {
+          "src": "banqup/logo.svg",
+          "alt": "banqup",
+          "width": 65,
+          "height": 65
+        }
+      }
+    }
+  },
+  "routes": [
+    {
+      "id": "default",
+      "vpDefinitionId": "banqup",
+      "steps": [
+        {
+          "id": "verifyRequest",
+          "operation": "navigate",
+          "path": "/credentials/verify/request",
+          "nextId": "infoRequest",
+          "isDefaultRoute": true
+        },
+        {
+          "id": "infoRequest",
+          "operation": "navigate",
+          "path": "/information/request",
+          "nextId": "credentialOffer"
+        },
+        {
+          "id": "credentialOffer",
+          "operation": "execute",
+          "action": "create-credential-offer",
+          "nextId": "issueRequest"
+        },
+        {
+          "id": "issueRequest",
+          "operation": "navigate",
+          "path": "/credentials/issue/request"
```

Comment:
check the knb demo and how we do redirects. something like:

  {
          "id": "issueRequest",
          "operation": "navigate",
          "path": "/credentials/issue/request",
          "nextId": "issueSuccess"
        },
        {
          "id": "issueSuccess",
          "operation": "navigate",
          "path": "https://knb.demo.sphereon.com"
        },

Reply:
Fixed